### PR TITLE
Design Happy Kitchen static website using index.html and style.css

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
First project using index.html and style.css to design a static website for Happy Kitchen.

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @TaiwoEnoch, :wave: 

Great job :+1: 
Your have done a great job implementing such a nice piece within a short period of time but there still some issues you need to work on before you can merge your project.



### Highlights
- Meaningful PR message :heavy_check_mark: 
- Great UI :heavy_check_mark: 

## Required Changes :recycle:
**Consider it a great investment to work on these features before merging:**
- [ ] Check the comments below to see some identified need for changes
- [ ] You have just a commit history related to this project. It is too small, and this sabotages the appealing effects of proper gitflow that allow visitors to track your progress while working on the project. > Please try to resolve thess changes in at least two separate commits.
- [ ] As indicated in your style.css, removing empty lines will make your code more DRY.
- [ ] in each of the page, please try to make the <kbd>ingredient -div</kbd> position at the center of the page

#### Note:
Do not close this pull request nor merge until it is resolved. You will use this same branch and request to implement the required changes.
_Don't hesitate to ask questions if there's any part that is not clear to you, please._

**Happy Coding!**

File: Recipe_1.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+<body>
+  <header>
+    <nav>
+     <a href="index.html" class="logo">Happy<span1>Kitchen</span></a>
+    </nav>
+    <ul class="navbar">
+      <li><a href="Recipe_1.html" class="nav-list">Pi<span1>zza</span1> |</a></li>
+      <li><a href="Recipe_2.html" class="nav-list"><span1>Bur</span1>ger |</a></li>
+      <li><a href="Recipe_3.html" class="nav-list">Sa<span1>lad</span1></a></li>
+    </ul>
```
Comment (AbayomiOlaoye) [inline Recipe_1.html]: Great work creating a nice navigation elements for easy site navigation. It's standard practice of HTML semantics to include all navigation links in your <code>nav</code> tag. Do you think you can replicate that too?

File: Recipe_1.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      <ul class="list-ingredient">
+        <li>1-1/2 cups (375 ml) warm water</li>
+        <li>1 package or 2-1/4 tsp (11.25 ml) active dry yeast</li>
+        <li>3-1/2 cups (875 ml) bread flour</li>
+        <li>2 Tbsp (30 ml) olive oil</li>
+        <li>2 tsp (10 ml) salt</li>
+        <li>1 tsp (5 ml) sugar</li>
+      </ul>
+    </nav>
+  </div>
```
Comment (AbayomiOlaoye) [inline Recipe_1.html]: Following standard practice of html semantics, <code>nav</code> can only be a parent to navigation links. Kindly remove the nav tag from this list.

File: Recipe_2.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        <li>1 or 2 cloves garlic, peeled and minced</li>
+        <li>About 1/2 teaspoon salt</li>
+        <li>About 1/4 teaspoon pepper</li>
+        <li>4 hamburger buns (4 in. wide), split</li>
+        <li>About 1/4 cup mayonnaise</li>
+        <li>About 1/4 cup ketchup</li>
+        <li>4 iceberg lettuce leaves, rinsed and crisped</li>
+        <li>1 firm-ripe tomato, cored and thinly sliced</li>
+        <li>4 thin slices red onion</li>
+      </ul>
```
Comment (AbayomiOlaoye) [inline Recipe_2.html:40]: Apply as required above.

Also, look at the display below, while you have done a great job hear, the div appears no to be well centered. Please fix that.

screenshot-127 0 0 1_5500-2022 11 17-12_05_32

File: style.css [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+.insight-words {
+  margin-left: 30px;
+  margin-bottom: 30px;
+  font-size: 20px;
+  color: #000;
+  padding: 20px;
+}
+
+
+
```
Comment (AbayomiOlaoye) [inline style.css]: Keeping codebase DRY improves its readability and understandability. Please avoid empty spaces or lines in your code. I think line 166 also needs some deletions.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+  color: #000;
+  padding: 18px;
+}
+
+.burger-procedures {
+  background-color:orange;
+}
+
+
+
```
Comment (AbayomiOlaoye) [inline style.css]: Here...

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]

**Review by AbayomiOlaoye — APPROVED** [review]
Congratulations @TaiwoEnoch :1st_place_medal: 

You are exceptional! Your project  is complete and there is nothing left than to merge it :clap: 

you_did

## Highlights
- [x] Good commit message
- [x] Implementation of required changes
- [x] DRY code

**Comment by TaiwoEnoch** [comment]
Thank you for being a brother indeed, sir.

**Comment by TaiwoEnoch** [comment]
It is nice being with you in a time like this.
Thanks so much for the brotherly love.

**Comment by TaiwoEnoch** [comment]
Yes, sir here is the link  to the pull request  of the hello microverse
project
https://github.com/TaiwoEnoch/Microverse_Hello/pull/1

I will come for more when I'm around tomorrow because I beep a little at
the first one we did I still need a bit of insight on it.
Thanks so much, sir.

**Pull request merged**